src/main.go

```diff
@@ -119,10 +119,10 @@ func main() {
 		fmt.Println("\nGGWP, you solved the puzzle!")
 	}
 
-	close(interruptChan)
```

Comment:
Move it just after the channel opening. Otherwise if an error occurs in `clearTerminal`, you will keep a channel opened even after the program close.